smoketests/config.toml

```diff
@@ -1,4 +1,4 @@
-default_server = "127.0.0.1:3000"
+default_server = "localhost"
```

Comment:
instead of using a hardcoded address, use a nickname for the server specified below. This allows us to look up the correct `protocol` for that server.